### PR TITLE
fix(frontend):Fixed contribution error page

### PR DESCRIPTION
**Pull request by nkrmr**
- Fixed error 500 pages when the contribution has no content
- Added contribution tests with and without external content

### CODE REVIEW

File: packages/code-du-travail-frontend/__tests__/contribution.test.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import { render } from "@testing-library/react";
+import FicheContribution from "../pages/fiche-ministere-travail/[slug]";
+
+describe("<FicheContribution />", () => {
+  it("should render with external content", () => {
+    const { container } = render(
+      <FicheContribution
+        query={{
+          slug:
+            "arret-maladie-pendant-la-periode-dessai-quelles-sont-les-regles"
```
Comment (lionelB) [inline packages/code-du-travail-frontend/__tests__/contribution.test.js:11]: I don't see the content in the snapshot, normal ?

File: packages/code-du-travail-frontend/pages/contribution/[slug].js [hunk 2]
```diff
@@ -28,11 +28,11 @@ class PageContribution extends React.Component {
...
 
-    const contentTag = markdown.match(contentRegExp)[0];
+    const contentTag = markdown.match(contentRegExp);
 
-    if (contentTag) {
+    if (contentTag && contentTag.length > 0) {
       // Extract URL from Content tag, only one for now
-      const contentUrl = contentTag.match(
+      const contentUrl = contentTag[0].match(
         /\bhttps?:\/\/(www\.)?[-a-zA-Z0-9@:%._\\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\\+.~#?&//=]*)/gi
```
Comment (lionelB) [inline packages/code-du-travail-frontend/pages/contribution/[slug].js:36]: this should be extracted into a function and battle tested

File: packages/code-du-travail-frontend/__tests__/contribution.test.js (hunk 1 above)
Comment (nkrmr) [reply packages/code-du-travail-frontend/__tests__/contribution.test.js:11]: Si il y est bien

File: packages/code-du-travail-frontend/__tests__/contribution.test.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import FicheContribution from "../pages/fiche-ministere-travail/[slug]";
+
+describe("<FicheContribution />", () => {
+  it("should render with external content", () => {
```
Comment (lionelB) [inline packages/code-du-travail-frontend/__tests__/contribution.test.js:6]: test should have unique name

**Review by lionelB — APPROVED** [review]

**Pull request merged**